src/components/sidenav/drawer.scss

```diff
@@ -0,0 +1,27 @@
+@import "variables";
+@import "shadows";
+
+
+:host {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: $z-index-drawer;
+  box-sizing: border-box;
+
+  background-color: white;
+  box-shadow: none;
+
+  will-change: transform;
+  transition: transform $swift-ease-out-duration $swift-ease-out-timing-function,
```

Comment:
FY: these animation timing functions need to be double-checked. They were originally copied from ng1 but they may be stale now.